docs/operation/bootstrap_backup_restore.rst

```diff
@@ -0,0 +1,49 @@
+Bootstrap node backup restore
+=============================
```

Comment:
minor: keep an empty line below `====` (to be consistent with what you do with `****`)